python/lsst/meas/deblender/deblend.py

```diff
@@ -276,7 +279,7 @@ def deblend(self, exposure, srcs, psf):
         n0 = len(srcs)
         nparents = 0
         for i, src in enumerate(srcs):
-            #t0 = time.clock()
+            # t0 = time.clock()
```

Comment:
`time.clock()` is deprecated so it's debatable whether this code should be here at all. See DM-15450

Reply:
Thanks for catching this. I suggest we fix it on DM-15450